spec/controllers/participants_controller_spec.rb

```diff
@@ -41,6 +42,38 @@
         }
       }.to change(Participant, :count).by(1)
     end
+
+    it "should assign the arm if there is only one arm on a protocol" do
+      attributes = @participant.attributes
```

Comment:
If you just want an attributes hash:

      create(:participant, ...) -> attributes_for(:participant, ...)